### ISSUE TITLE
Práce s externími identifikátory

### ISSUE THREAD

**Issue by RAD-X** [Spoje-NET/FlexiPeeHP#16]
Tento kód sice funguje (kupodivu - haluz vs úmysl? :-) ) ale háže warning pro rawurlencode protože je předána array namisto stringu.
```
$save = array(
	'id' => array(
		$addr->id, // timto identifikujeme co se ma upravovat
		// pridany externi identifikator 
		$external_id
	),
	'eanKod' => $member_id
);
$adresar = new \FlexiPeeHP\Adresar($save, $this->make_conf());
$savedata = $adresar->insertToFlexiBee();
```
samozřejmě já to řešil raději takto
```
$ids = array(
	$addr->id, // timto identifikujeme co se ma upravovat
	// pridany externi identifikator 
	$external_id
);
$ids = json_encode($ids);
$save = array(
	'id' => $ids,
	'eanKod' => $member_id
);
$adresar = new \FlexiPeeHP\Adresar($save, $this->make_conf());
$savedata = $adresar->insertToFlexiBee();
```

Chvíli jsem nad tím musel přemýšlet jak to správně vložit protože jsem nenašel žádnou dokumentaci ani příklad a metoda/funce setMyKey ve FlexiBeeRO evidentně není určená k přiřazování externích identifikátorů - resp. je nefunkční v mém pojetí.
Nebo je na toto nějaký jiný přístup?

Jinak díky za tuto library.

**Comment by RAD-X** [comment]
Ještě jsem si našel chvilku a vidím že jsem přehlédl ve FlexibeeRW funkce changeExternalID a addExternalID sic je to oproti mému příkladu o jedno API volání navíc, ale metody tu na to jsou. 😃 

Takže vlastně k úplné dokonalosti by bylo fajn automaticky skládat/encodovat položky parametrů pokud nejsou typu string tak aby nevznikalo PHP varování které je zmíněné v mém předešlém příspěvku.

**Comment by Vitexus** [comment]
Ano, tak jak navrhujete v předchozím příspěvku jsem chtěl k externím id přistupovat ale nakonec mě nic nedonutiolo tento mechanizmus implementovat.

**Comment by Vitexus** [comment]
Zatím jsem tedy podniknul první krok k tomu aby bylo možné co navrhujete:
https://github.com/Spoje-NET/FlexiPeeHP/commit/a6a698586e5ef169f42112f436f56daace410a5a
